spec/system/manage_maps_spec.rb

```diff
@@ -1,10 +1,12 @@
 require "rails_helper"
 
 RSpec.describe "manage maps", type: :system do
-  it "lists maps" do
+  it "lists maps when no map selected" do
```

Comment:
Do we want anything to ensure the map drawer is closed when a map is selected?

Reply:
All of the other specs that have a map selected need to call the step that opens the map selector in order to pass. Since the default state is to have it collapsed and all the existing specs would break if that were to change, I'm comfortable with only having an additional explicit spec for this behavior.